vtkext/private/module/vtkF3DConsoleOutputWindow.h

```diff
@@ -32,7 +44,6 @@ class vtkF3DConsoleOutputWindow : public vtkOutputWindow
   vtkF3DConsoleOutputWindow();
   ~vtkF3DConsoleOutputWindow() override = default;
 
-private:
```

Comment:
why not private ?

Reply:
I need to access to `UseColoring` in my subclass.

Reply:
use vtkSet/GetMacro instead, its better to follow VTK standard for our VTK classes